HANDBOOK.md

```diff
@@ -97,18 +98,43 @@ The config file consists of a handful of different indexes.
 
 Unfortunately. we sometimes need to manually change a type definition, albeit rarely. The `typeOverride.json` file allows us to overwrite any updates the script attempts to make that we don’t want to happen.
 
-🛠 _At the moment, updating the registry is a manual process when it should be something that runs after a major release automatically. We are investigating how to best make this happen as of 7/01/2021._
-
 🛠 _Another issue is we are limited by the permissions and licenses of the org that we are running the update script on, which may return incomplete describe information. We need to address this as soon as possible to not run into type gaps between releases. This is being worked on as of 7/16/2021._
 
+### Breaking Down a Metadata Type Entry
+
+Luckily we're able to automatically generate most metadata type entries without any human interaction. However, we sometimes get it wrong, or we can't account for a setting. When this happens, we'll need to manually update the registry to get the desired behavior.
+
+There's a lot of options, and toggles that will affect what happens to a metadata type, below, we'll explain what each does, and more importantly, when you'd want to enable each one.
+
+In SDR, a metadata type entry is required to have, at minimum, 2 values, the `id` and the `name`
+
+- `id` The `id` is a unique identifier for the metadata type. It's usually the API name lowercased. For `ApexClass` this becomes, `apexclass`
+- `name` The `name` is the API name of the metadata type, as you would see it, so `ApexClass`.
+
+Now, for all the optional properties which define how the metadata type is converted, where it's stored, and how we can recognize it.
+
+- `directoryName` The `directoryName` is the name of the directory where components are located in a package. Continuing with the `ApexClass` type, this would be `classes`.
+- `suffix` The `suffix` is the suffix of the metadata type file, which is also in the `suffixes` entry in the registry. You might be wondering why is this an optional property? Well, some types, such as `LightningComponentBundles` and `Documents` don't require a certain suffix, or are a "bundle" type which is made of multiple files and directories. For ApexClass, this is `.cls`
+- `strictDirectoryName` The `strictDirectoryName` is a boolean that determines whether components MUST reside in a folder named after the type's `directoryName`
+- `ignoreParsedFullName` This is a boolean that dictates whether to ignore the `fullName` that's parsed from the file path. If set to `true` the metadata type's name will be used instead. This is set to true for `CustomLabels`
```

Comment:
@WillieRuemmele would it be possible to either provide a link or a glossary in the handbook to explain things like "fullName?"